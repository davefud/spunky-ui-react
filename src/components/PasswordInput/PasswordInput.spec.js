import React from 'react';
import renderer from 'react-test-renderer';
import PasswordInput from './PasswordInput';

test('hides password quality by default', () => {
    const tree = renderer.create(<PasswordInput
        htmlId='test'
        name='test'
        onChange={() => {}}
        value='IUacac#7wfe' />).toJSON();
    expect(tree).toMatchSnapshot();
});