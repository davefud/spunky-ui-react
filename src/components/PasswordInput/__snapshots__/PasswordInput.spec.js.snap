// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hides password quality by default 1`] = `
<div
  style={
    Object {
      "marginBottom": 16,
    }
  }
>
  <label
    htmlFor="test"
    style={
      Object {
        "display": "block",
      }
    }
  >
    Password
     
    <span
      style={
        Object {
          "color": "red",
        }
      }
    >
       *
    </span>
  </label>
  <input
    id="test"
    maxLength={50}
    name="test"
    onChange={[Function]}
    type="password"
    value="IUacac#7wfe"
  />
</div>
`;
